dev_tools/scripts/db_integrity_test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Insert some test data in the db
+docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submission_file_path,submission_user,submission_file_extension,status,encryption_method) VALUES('test.c4gh','dummy','c4gh','INIT','CRYPT4GH');"
+
+# Build, run and execute the pg_basebackup action
+docker container run --rm -i --name pg-backup --network=host $(docker build --build-arg USER_ID=$(id -u) -f dev_tools/Dockerfile-backup -q -t backup .) /bin/sda-backup --action pg_basebackup
+
+# Find the name of the copy in the S3
+DBCOPY=$(s3cmd -c dev_tools/s3conf ls s3://dumps/ | grep ".tar" | cut -d '/' -f4)
+
+# Checks if the backup image exists locally from the previou step, if not creates the image
+# Then gets the physical copy of the database from the S3
+IMAGENAME=$(docker images -q backup)
+NAMELENGTH=`echo -n "$IMAGENAME" | wc -m`
+
+if [ $NAMELENGTH = 0 ]; then
+    docker container run -v $(pwd)/tmp:/home --rm -i --name pg-backup --network=host $(docker build --build-arg USER_ID=$(id -u) -f dev_tools/Dockerfile-backup -q .) /bin/sda-backup --action pg_bb-restore --name "$DBCOPY"
+else
+    docker container run -v $(pwd)/tmp:/home --rm -i --name pg-backup --network=host backup /bin/sda-backup --action pg_bb-restore --name "$DBCOPY"
+fi
```

Comment:
For this scenario to hold true the build on line 7 needs to fail.
And that means that $DBCOPY will be empty. (L10)

Reply:
True. I do not know what is wrong with me sometimes :)
Fixed in the last commit